src/components/Overview/ProductInfo.jsx

```diff
@@ -16,7 +17,7 @@ const ProductionInfo = ({productDetails, currentStyle, ratings, numOfReviews}) =
       href='#RatingsAndReviews'
       style={{display: numOfReviews === 0 ? 'none' : ''}}
     >
-      Read All {numOfReviews} Reviews
+      <Span>Read All {numOfReviews} Reviews</Span>
```

Comment:
The text in here needs to be in quotes so that num of reviews dynamically renders (also missing a $)